aicsimageio/tests/readers/test_czi_reader.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import xml.etree.ElementTree as ET
+from typing import List, Tuple
+
+import numpy as np
+import pytest
+
+from aicsimageio import exceptions
+from aicsimageio.readers import ArrayLikeReader, CziReader
+
+from ..conftest import LOCAL, get_resource_full_path
+from ..image_container_test_utils import (
+    run_image_container_mosaic_checks,
+    run_image_file_checks,
+)
+
+# from aicsimageio.readers.default_reader import DefaultReader
+
+
+@pytest.mark.parametrize(
+    "filename, "
+    "set_scene, "
+    "expected_scenes, "
+    "expected_shape, "
+    "expected_dtype, "
+    "expected_dims_order, "
+    "expected_channel_names, "
+    "expected_physical_pixel_sizes",
+    [
+        (
+            "s_1_t_1_c_1_z_1.czi",
+            "Scene:0",
+            ("Scene:0",),
+            (1, 325, 475),
+            np.uint16,
+            "CYX",
+            ["Channel:0--Bright--Brightfield"],
+            (1.0, 1.0833333333333333e-06, 1.0833333333333333e-06),
+        ),
+        (
+            "s_3_t_1_c_3_z_5.czi",
+            "P2",
+            ("P2", "P3", "P1"),
```

Comment:
Because there are multiple scenes in this file you should write three tests and select different scenes in each one.